namui/src/namui/system/font/mod.rs

```diff
@@ -0,0 +1,68 @@
+use crate::{
+    namui::{skia::Font, FontType, TypefaceType},
+    Typeface,
+};
+use dashmap::DashMap;
+use std::sync::Arc;
+
+use super::InitResult;
```

Comment:
```suggestion
use super::InitResult;
use crate::{
    namui::{skia::Font, FontType, TypefaceType},
    Typeface,
};
use dashmap::DashMap;
use std::sync::Arc;
```

This is lint result after removing empty line between imports